examples/image_classification/utils.js

```diff
@@ -1,3 +1,4 @@
+let _isQuantized = false;
```

Comment:
@mingmingtasd  Please also update others examples' utils.js files, thanks

Reply:
Same comments as BruceDai said, i just checked https://github.com/intel/webml-polyfill/blob/master/examples/object_detection/utils.js

Reply:
I suggest passing this param at here.
https://github.com/intel/webml-polyfill/blob/1045b3cd712a8fe3d405ccba4ac8dfe085418cb8/examples/image_classification/utils.js#L101 to avoid duplication of work

Reply:
@ibelem @BruceDai @pinzhenx Thanks for your suggestions. I updated codes, plz check.